solr/modules/monitor/src/java/org/apache/lucene/monitor/MonitorFields.java

```diff
@@ -0,0 +1,38 @@
+/*
+ *
+ *  * Licensed to the Apache Software Foundation (ASF) under one or more
+ *  * contributor license agreements.  See the NOTICE file distributed with
+ *  * this work for additional information regarding copyright ownership.
+ *  * The ASF licenses this file to You under the Apache License, Version 2.0
+ *  * (the "License"); you may not use this file except in compliance with
+ *  * the License.  You may obtain a copy of the License at
+ *  *
+ *  *     http://www.apache.org/licenses/LICENSE-2.0
+ *  *
+ *  * Unless required by applicable law or agreed to in writing, software
+ *  * distributed under the License is distributed on an "AS IS" BASIS,
+ *  * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ *  * See the License for the specific language governing permissions and
+ *  * limitations under the License.
+ *
+ */
+
+package org.apache.lucene.monitor;
+
+import java.util.Set;
+
+public class MonitorFields {
+
+  public static final String QUERY_ID = QueryIndex.FIELDS.query_id + "_";
+  public static final String CACHE_ID = QueryIndex.FIELDS.cache_id + "_";
+  public static final String MONITOR_QUERY = QueryIndex.FIELDS.mq + "_";
+  public static final String PAYLOAD = QueryIndex.FIELDS.mq + "_payload_";
+  public static final String VERSION = "_version_";
+  public static final String ANYTOKEN_FIELD = TermFilteredPresearcher.ANYTOKEN_FIELD;
```

Comment:
cross-reference https://github.com/apache/lucene/pull/13379 w.r.t. making `TermFilteredPresearcher.ANYTOKEN_FIELD` public

Reply:
added https://github.com/apache/solr/pull/2382/commits/28bf2641840f0dfe8f1076661e3387cb03135f7e to use `TermFilteredPresearcher.ANYTOKEN_FIELD`